src/lib/components/docs/api-toggle.svelte

```diff
@@ -0,0 +1,98 @@
+<script lang="ts">
+  // Couldn't create this component entirely dynamic, because Slots can't be named dynamically
+  // Had to use !important to make sure the styles from tailwinds prose-class are overridden
+  import type { comparisonItem } from "$lib/types/docs";
+
+  export let items: comparisonItem[] = [
+    {
+      mobileTitle: "cURL",
+      title: "cURL",
+      value: 1,
+      slotName: "curlExample",
+    },
+    {
+      mobileTitle: "Go",
+      title: "Go ",
+      value: 2,
+      slotName: "goExample",
+      hidden: false,
+    },
+    {
+      mobileTitle: "Example Response",
+      title: "Example Response",
```

Comment:
Could we swap to simply response? I suppose example is implied by the data.

```suggestion
      mobileTitle: "Response",
      title: "Response",
```

Reply:
I'm also torn that this should be in a tab. I almost feel like I'd want to see this information by default, also.